svc-device-manager/rest/redfish/client.go

```diff
@@ -143,6 +143,39 @@ func (h *HttpClient) Post(uri string, requestBody []byte) (*http.Response, error
 	return resp, nil
 }
 
+func (h *HttpClient) Patch(uri string, requestBody []byte) (*http.Response, error) {
```

Comment:
Please add a comment what this func do (like in previous functions in this file)

Reply:
Done.